__tests__/Unit/Components/ProgressForm/ProgressForm.test.tsx

```diff
@@ -16,7 +18,11 @@ const mockQuestion = [
 
 describe('Progress form', function () {
```

Comment:
line renderWithRouter(.... This line is calling the renderWithRouter function, which renders the component in the context of a router. However, the ProgressForm component does not need to be rendered in the context of a router. Therefore, you can remove the renderWithRouter function call.

Reply:
Since i am getting the task id from route parameter so I am calling on useRouter in progress Form component, thus I have rendered it with router